README.md

```diff
@@ -1,2 +1,65 @@
-# worldcovid
-Web app that display worldwide stats about COVID19.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# COVID19 World Stats
+Covid19 World Stats is a web app that displays stats about COVID19 worldwide.
+
+It also allows the user to check for detailed information of each country.
+
+This project was created in order to practice React and Redux.
+
+### Preview
+
+![](./public/preview.gif)
+
+## Live Version
+
+[Visit live version](https://covid19worldwidestats.netlify.app)
+
+## Built With
+- HTML, CSS, JavaScript
+- React
+- Redux
+- Webpack
+- Babel
+
+## Getting Started
+- Follow setup instructions below
+
+
+### Prerequisites
+
+- node.js
+- Javascript
+- Git and Github
+- Terminal
+- browser
+
+### Setup
+
+- clone the following repo `git@github.com:orozCoding/worldcovid.git`
+- cd into the directory and run npm install to install dependencies
+- run npm start
+
+### Usage
+- Run npm install on a Terminal to install the modules
+- run npm start
```

Comment:
- [ ] [OPTIONAL] **GOOD JOB 🎉🎊🥳**. The app is working well and to improve your readme I'd suggest wrapping npm start, npm install into code like `npm start`, `npm install` using ` ``  `. Because if there are many commands it is easy to copy from it instead of typing one by one 🙂😉👍